src/dvclive/live.py

```diff
@@ -269,6 +269,11 @@ def log_image(self, name: str, val):
         if not Image.could_log(val):
             raise InvalidDataTypeError(name, type(val))
 
+        if isinstance(val, str) or isinstance(val, Path):
+            from PIL import Image as ImagePIL
+
+            val = ImagePIL.open(val)
+
```

Comment:
So we end up making our own copy of these images inside `dvclive/plots/images`, right? Should we leave them as is and add `plots: [original/path/to/image/file]` in `dvclive/dvc.yaml`?

Not that strong an opinion here but want to hear your thoughts.

Reply:
> Should we leave them as is and add plots: [original/path/to/image/file] in dvclive/dvc.yaml?

I think in this case we can expect that user take care of them on their own (e.g. in the `example-get-started` repo). Also, it might be another option for this, yes. I think can have both eventually. `dvc.yaml` might become very complicated though ...

In this specific case, the YOLO repo creates images in some temporary directory specific to an experiment and we need to capture it from that directory. Ideally, the repo itself should be using `log_image` instead of creating them the regular way, but I don't think it's realistic / practical to expect them to convert it soon. So for now, to unblock it and show the benefits I think it's a reasonable approach.